content/2022-munich/schedule.md

```diff
@@ -0,0 +1,361 @@
+---
+title: Schedule
+---
+
+## PromCon 2022 Schedule
+
+<table class="table schedule-table">
+  <tr class="day">
+    <th colspan="3">Day 1 - Tuesday, November 8</th>
+  </tr>
+  <tr class="break">
+    <td>8:15</td>
+    <td>Breakfast and Registration</td>
+    <td></td>
+  </tr>
+   <tr class="talk">
+    <td>9:00</td>
+    <td>
+      Welcome
+    </td>
+    <td>
+      <a href="../speakers/richard-richih-hartmann">Richard RichiH Hartmann</a>
```

Comment:
For RichiH, we probably want to add the quotes around the RichiH again, like in his bio: `Richard "RichiH" Hartmann`